xapian-letor/scorer/err_score.cc

```diff
@@ -72,9 +71,9 @@ ERRScore::score(const std::vector<FeatureVector> & fvv) const
 	 * http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.74.9057&rep=rep1&type=pdf
 	 */
 	double relevance_probability = (exp2(fvv[rank - 1].get_label()) -
-					 1) / max_value;
+		1) / max_value;
```

Comment:
I think this was better before.

Though probably more readable to use an intermediate variable, and so avoid needing to wrap the expression:

```
 	auto label = fvv[rank - 1].get_label();
 	double relevance_probability = (exp2(label) - 1) / max_value;
```